Firmware/ultralcd.cpp

```diff
@@ -627,8 +627,8 @@ void lcdui_print_status_line(void) {
 //! @endcode
 void lcdui_print_status_screen(void)
 {
-
-    lcd_set_cursor(0, 0); //line 0
+    lcd_frame_start();
```

Comment:
Do we need `lcd_frame_start()` here? I think `lcd_update()` takes care of it now.

Reply:
Hmmm. Maybe this will work. I need to test this.

Reply:
I thought about this more and I think I don't want to remove this. Thing is that for the status screen, the text on it can change even without interacting with the knob (take status message updates for example). So it makes sense that a full redraw might start even without interacting with the printer. This is in contrast to menus where only interacting with them will cause a redraw. So I'll keep this here

Reply:
`lcd_updatestatus()` sets `lcd_draw_update = 1;` This means `menu_lcd_lcdupdate_func` should be calling `lcd_frame_start()` before rendering the status screen:

```
if (lcd_draw_update) {
    lcd_timeoutToStatus.start();
    lcd_frame_start();
}
(*menu_menu)();
```

From what I can find in Vscode, it looks like `lcd_status_screen` is always called as a menu.

Reply:
Only exception I can find is my hack in the MMU code, to render the full white blocks when LCD updates are disabled.

```c
void TryLoadUnloadReporter::Render(uint8_t col, bool sensorState) {
    // Set the cursor position each time in case some other
    // part of the firmware changes the cursor position
    lcd_insert_char_into_status(col, sensorState ? LCD_STR_SOLID_BLOCK[0] : '-');
    if (!lcd_update_enabled) lcdui_print_status_line();
}
```

Reply:
That is not the only time the status screen changes. A change in displayed data can also happen based on the printing status (say SD printing or USB printing or idle, etc...). So I prefer to have an explicit frame start for the status screen

Reply:
Marking this as resolved since there is no risk involved.